Lib/test/test_regrtest.py

```diff
@@ -0,0 +1,427 @@
+"""
+Tests of regrtest.py.
+
+Note: test_regrtest cannot be run twice in parallel.
+"""
+from __future__ import print_function
+
+import collections
+import os.path
+import platform
+import re
+import subprocess
+import sys
+import sysconfig
+import tempfile
+import textwrap
+import unittest
+from test import support
+
+
+Py_DEBUG = hasattr(sys, 'getobjects')
+ROOT_DIR = os.path.join(os.path.dirname(__file__), '..', '..')
+ROOT_DIR = os.path.abspath(os.path.normpath(ROOT_DIR))
+
+TEST_INTERRUPTED = textwrap.dedent("""
+    from signal import SIGINT
+    try:
+        from _testcapi import raise_signal
+        raise_signal(SIGINT)
+    except ImportError:
+        import os
+        os.kill(os.getpid(), SIGINT)
+    """)
+
+
+SubprocssRun = collections.namedtuple('SubprocssRun', 'returncode stdout, stderr')
```

Comment:
Oh wow, I didn't notice that I did two mitakes: typo in the type name, and "," typo in the string describing fields. I'm surprised because "," seems to be simply ignored!? Fixed in PR 1516.

```
>>> collections.namedtuple('Point', 'x,y z')._fields
('x', 'y', 'z')
```

It seems like namedtuple tries to be kind and fix my mistakes :-)